content/docs/0.6.0/reference/monitoring/dynatrace/index.md

```diff
@@ -52,55 +35,35 @@ Please make sure to have the following tool(s) installed:
 
-1. Clone the install repository and setup your credentials by executing the following steps:
-  ```console
-  git clone --branch 0.5.0 https://github.com/keptn-contrib/dynatrace-service --single-branch
-  ```
-  ```console
-  cd dynatrace-service/deploy/scripts
-  ```
-  ```console
-  ./defineDynatraceCredentials.sh
-  ```
-    When the  script asks for your Dynatrace tenant, please enter your tenant according to the appropriate pattern:
-      - Dynatrace SaaS tenant: `{your-environment-id}.live.dynatrace.com`
-      - Dynatrace-managed tenant: `{your-domain}/e/{your-environment-id}`
-
-1. Execute the installation script for your platform:
-
-  - If you are on **Azure AKS**, please execute
+1. Store your credentials in a Kubernetes secret by executing the following command. The `DT_TENANT` has to be set according to the appropriate pattern:
+  - Dynatrace SaaS tenant: `{your-environment-id}.live.dynatrace.com`
+  - Dynatrace-managed tenant: `{your-domain}/e/{your-environment-id}`
 
     ```console
-    ./deployDynatraceOnAKS.sh
+    kubectl -n keptn create secret generic dynatrace --from-literal="DT_API_TOKEN=<DT_API_TOKEN>" --from-literal="DT_TENANT=<DT_TENANT>" --from-literal="DT_PAAS_TOKEN=<DT_PAAS_TOKEN>"
     ```
 
-    - If you are on **AWS EKS**, please execute
-
-    ```console
-    ./deployDynatraceOnEKS.sh
-    ```
+1. Get the latest *dynatrace-service* version and deploy it: 
 
-  - If you are on **Google GKE**, please execute
+    **Attention:** There is no official release of the dynatrace-service yet. It will be available with the Keptn 0.6.0 release, which is planned for mid of January 2020. For an experimental try, you can use the master branch as shown below:
 
     ```console
-    ./deployDynatraceOnGKE.sh
+    git clone --branch master https://github.com/keptn-contrib/dynatrace-service --single-branch
     ```
-        
-    Please read **Note 2** after this section in case you are running GKE Container-optimized os.
-
-  - If you are on **Pivotal PKS**, please execute
 
     ```console
-    ./deployDynatraceOnPKS.sh
+    kubectl apply -f deploy/manifests/dynatrace-service/dynatrace-service.yaml
     ```
 
-  - If you are on **OpenShift**, please execute
+1. When the service is deployed, use the following command to install Dynatrace on your cluster. If Dynatrace is already deployed, the current deployment of Dynatrace will not be modified.
 
     ```console
-    ./deployDynatraceOnOpenshift.sh
+    keptn configure monitoring dynatrace
     ```
 
-When this script is finished, the Dynatrace OneAgent and the *dynatrace-service* are deployed in your cluster. Execute the following commands to verify the deployment of the OneAgent as well as of the *dynatrace-service*.
+**Verify deployment in your cluster**
+
+When keptn configure monitoring is finished, the Dynatrace OneAgent is deployed in your cluster. Execute the following commands to verify the deployment of the OneAgent as well as of the *dynatrace-service*:
```

Comment:
Use source formatting for `keptn configure monitoring`?

Reply:
Thx, the command is now properly formatted.